modules/pulsar/src/main/java/org/testcontainers/containers/PulsarContainer.java

```diff
@@ -92,7 +92,12 @@ protected void setupCommandAndEnv() {
 
         List<WaitStrategy> waitStrategies = new ArrayList<>();
         waitStrategies.add(Wait.defaultWaitStrategy());
-        waitStrategies.add(Wait.forHttp(METRICS_ENDPOINT).forStatusCode(200).forPort(BROKER_HTTP_PORT));
+        waitStrategies.add(
+            Wait
+                .forHttp(ADMIN_CLUSTERS_ENDPOINT)
+                .forPort(BROKER_HTTP_PORT)
+                .forResponsePredicate("[\"standalone\"]"::equals)
```

Comment:
will this work if someone wants to start a cluster of Pulsar containers? Is there something else we can check to be safe?

Reply:
No, but the whole container is currently set up around running a standalone cluster. If you actually wanted to run a cluster the command would need a rethink.